examples/route_guide/routeguide/route_guide.pb.go

```diff
@@ -14,8 +14,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.32.0
-// 	protoc        v4.25.2
+// 	protoc-gen-go v1.33.0
+// 	protoc        v3.21.12
```

Comment:
Yeah, I'd chosen not to care about this because I wasn't sure if we were going to be landing the updated example generated code at all (see my note at the bottom of the PR description). I've now regenerated all of the examples using protoc v4.25.2.